airflow/www/views.py

```diff
@@ -920,10 +924,16 @@ def clear(self):
         dag_id = request.args.get('dag_id')
         task_id = request.args.get('task_id')
         origin = request.args.get('origin')
-        dag = dagbag.get_dag(dag_id)
+        orm_dag = models.DagModel.get_dagmodel(dag_id)
+        if not orm_dag:
+            flash("Dag '{}' does not exist".format(dag_id), 'error')
```

Comment:
nitpick for consistency:

```suggestion
            flash("DAG '{}' does not exist".format(dag_id), 'error')
```

Reply:
fixed